src/gui/optionsdlg.h

```diff
@@ -60,6 +60,8 @@ namespace Ui
 class OptionsDialog: public QDialog
 {
     Q_OBJECT
+    using ThisType = OptionsDialog;
```

Comment:
Yes, I do. And I like C++ for the fact that the only difference between those two is whether my type fits into a CPU register or not.